WordPress/src/main/java/org/wordpress/android/models/SiteSettingsModel.java

```diff
@@ -15,57 +15,54 @@
  */
 
 public class SiteSettingsModel {
-    public static final int RELATED_POSTS_ENABLED_FLAG = 0x1;
-    public static final int RELATED_POST_HEADER_FLAG = 0x2;
-    public static final int RELATED_POST_IMAGE_FLAG = 0x4;
+    private static final int RELATED_POSTS_ENABLED_FLAG = 0x1;
```

Comment:
Thanks for fixing these! I'd suggest doing them in a separate PR though in the future, so it's easier to review. I do make this "mistake" quite often as well :(